### PR TITLE
make system serializer public

### PR DESCRIPTION
**Pull request by sclasen**
@dpiddy im +1 to merge this

### CODE REVIEW

**Comment by dpiddy** [comment]
Think this should include anything else? Like defaulting `SystemSerializer` to `Serializer` if not set?

**Comment by sclasen** [comment]
Yep should prob just default to json.

**Comment by dpiddy** [comment]
@sclasen updated

**Comment by sclasen** [comment]
@dpiddy after thinking a bit Im not positive defaulting to `Serializer` is the safest thing. lets say someone was somehow using the ProtobufSerializer today. upgrading to this would break things since there arent protobuf defs for the system structs.

I think we should keep the if nil SystemSerializer = Json in Init.

File: activity/worker.go [hunk 1]
```diff
@@ -81,8 +81,8 @@ func (a *ActivityWorker) Init() {
 		a.Serializer = fsm.JSONStateSerializer{}
 	}
 
-	if a.systemSerializer == nil {
-		a.systemSerializer = fsm.JSONStateSerializer{}
+	if a.SystemSerializer == nil {
+		a.SystemSerializer = a.Serializer
```
Comment (sclasen) [inline activity/worker.go]: this should default to fsm.JSONStateSerializer{}

**Comment by dpiddy** [comment]
Yeah, that sounds good.

**Comment by dpiddy** [comment]
Dropped the second commit, back to just making SystemSerializer public.

**Comment by sclasen** [comment]
:+1:

**Pull request merged**